_announcements/2019-05-27-klahanie-graduating-seniors.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+title: Klahanie Graduating Seniors
+date: 2019-05-27 08:20
+author: Klahanie
+comments: true
+categories: [announcements]
+show-title: true
+---
+http://klahanie.com/wp-content/uploads/2019/05/2019-Graduation-Clip-Art.jpg

```

Comment:
I'm assuming these announcements are placeholders for future content, but I wanted to mention that this url is showing on the webpage, as is the "</div> below. Also, this page seems like it's using a different font (size) for the body text than the other Announcements pages.